app/helpers/application_helper.rb

```diff
@@ -1,2 +1,18 @@
 module ApplicationHelper
+  def bootstrap_class_for flash_type
+    { success: "alert-success", error: "alert-danger", alert: "alert-warning", notice: "alert-info" }[flash_type.to_sym] || flash_type.to_s
+  end
+
+  def flash_messages(opts = {})
+    flash.each do |msg_type, message|
+      concat(content_tag(:div, message, class: "alert #{bootstrap_class_for(msg_type)} alert-dismissible", role: 'alert') do
+        concat(content_tag(:button, class: 'close', data: { dismiss: 'alert' }) do
+          concat content_tag(:span, '&times;'.html_safe, 'aria-hidden' => true)
```

Comment:
Re Austin's comments about not merging into master - the tests all work for me. Are they breaking for you? The branch is entirely functional for the user on my end. I wanted to merge to master so that if any of you were working on bootstrap display things today, or noticed that the code on master doesnt capitalize or link things to each other, and you decided you wanted to make those changes, that we dont do double work. I think it's really important to have up-to-date code so we're not re-doing the same thing and everyone is pulling down regularly. I think we might be going too long between merges. Thoughts?